src/components/sections/employeeHighlight/employeeHighlight.module.css

```diff
@@ -0,0 +1,69 @@
+.wrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 0 2rem;
+  max-width: 64rem;
```

Comment:
I know that the design specifies a width of 64rem, but for the rest of the site we have used 960px for the max-width, as 64rem will be relatively wide on smaller laptop screens 😅

Reply:
I'm seeing multiple pages with max-width set to [1400px](https://github.com/varianter/variant.no/blob/v3-petterhh/employee-highlight/src/components/sections/employees/employees.module.css#L9), [1200px](https://github.com/varianter/variant.no/blob/v3-petterhh/employee-highlight/src/components/employeePage/employeePage.module.css#L10), [1000px](https://github.com/varianter/variant.no/blob/v3-petterhh/employee-highlight/src/components/sections/imageSection/imageSectionComponent.module.css#L3) 🤔

Reply:
Yeah the maxwidth for the page is different from the maxwidth of some components, so some components we want to take the whole width of the screen e.g. the employee-section on the people-page or some images on the customer case page, whereas other places we want some components to be smaller than the width of the page, like the project info section on the customer case page. Until now I have used a max-width of 960px for all components with width 68rem in Figma, mostly because that was used more in the beginning, but also that the difference between sections that were supposed to be full-width vs. not full-width was almost non-existent on screensizes like 1024px when using 68rem. 

This might be a larger discussion though to ensure consistent use of widths across the page!

Reply:
You can leave it for now and we can do a full clean-up either going for 68rem or 960px as we see fit later 🤓

Reply:
Yeeeah, that sounds like a wider discussion indeed 😅